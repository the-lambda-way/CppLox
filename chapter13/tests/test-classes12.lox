class Foo {
  init() {
    print this;
  }
}

var foo = Foo();
print foo.init();
