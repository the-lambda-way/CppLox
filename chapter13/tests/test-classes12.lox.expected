Foo instance
Foo instance
Foo instance
